widgetssdk/src/main/java/com/glia/widgets/core/screensharing/ScreenSharingControllerImpl.kt

```diff
@@ -43,31 +53,27 @@ internal class ScreenSharingControllerImpl(
         }
     }
 
-    override fun onScreenSharingStarted() {
-        Logger.d(TAG, "screen sharing started")
+    private fun onScreenSharingStarted() {
         viewCallbacks.forEach { it.onScreenSharingStarted() }
     }
 
-    override fun onScreenSharingEnded() {
-        Logger.d(TAG, "screen sharing ended")
+    private fun onScreenSharingEnded() {
```

Comment:
Minor question. Maybe it makes sense to leave these debug logs for screen sharing started/ended events? It was useful for debugging. But it's not a big deal to add them if needed, so I don't insist, just asking your opinion and trying to understand why you decided to remove.

Reply:
These logs are duplicates since they are already logged where events are handled in the `EngagementRepositoryImpl`.

Reply:
I see, okay, this is a good change then. 👍